prov/efa/test/efa_unit_test_info.c

```diff
@@ -876,34 +876,6 @@ void test_efa_nic_select_first_device_with_surrounding_comma_matches() {
 	free(nic_name);
 }
 
-/**
```

Comment:
I am wondering why the other `test_efa_nic*` unit tests are still working. Is it false positive?

Reply:
Then the tests actually lose their value now?

Reply:
Good point. Let me move the `all` and multiple NIC selection tests to fabtests as well